github/ipld.yml

```
@@ -1815,6 +1815,21 @@ teams:
         - laurentsenta
     parent_team_id: w3dt-stewards
     privacy: closed
+  ipfs-gui:
+    members:
+      maintainer:
+        - rvagg
+        - sgtpooki
+        - tinytb
+      member:
+        - djmcquillan
+        - hacdias
+        - juliaxbow
+        - lidel
+        - olizilla
+        - sgtpooki
+        - whizzzkid
```

These users are not org members. You should either remove them from the team or add them to org members.

```suggestion
      maintainer:
        - rvagg
      member:
        - hacdias
        - lidel
        - olizilla
```

Things like these should be flagged automatically. This is tracked in https://github.com/protocol/github-mgmt-template/issues/93 and https://github.com/protocol/github-mgmt-template/issues/50

---

I added the members to the ipld organization. Does that remove the need to implement your suggested changes in this comment?

---

```suggestion
      maintainer:
        - rvagg
        - sgtpooki
        - tinytb
      member:
        - djmcquillan
        - hacdias
        - juliaxbow
        - lidel
        - olizilla
        - whizzzkid
```

---

Sure, I just removed the 1 duplicate entry.